G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookMoveToCommand.md

```diff
@@ -0,0 +1,40 @@
+# outlook.moveto
+
+## Syntax
+
+```G1ANT
+outlook.moveto item ⟦outlookmail/outlookfolder⟧ destinationfolder ⟦outlookfolder⟧
+```
+
+## Description
+
+This command is used to move an individual email message or a whole folder to another location (Outlook folder).
+
+| Argument            | Type                                                         | Required | Default Value                                                | Description                                                  |
+| ------------------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `item`              | [outlookmail](../../../g1ant.language/structures/outlookmailstructure.md) or [outlookfolder](../../../g1ant.language/structures/outlookfolderstructure.md) | yes      |                                                              | An item (a message or a folder) to be moved                  |
+| `destinationfolder` | [outlookfolder](../../../g1ant.language/structures/outlookfolderstructure.md) | yes      |                                                              | Destination Outlook folder                                   |
+| `if`           | [bool](../../../g1ant.language/structures/booleanstructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](../../../g1ant.language/structures/timespanstructure.md) | no       | [♥timeoutcommand](../../../appendices/special-variables.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](../../../g1ant.language/structures/procedurestructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](../../../g1ant.language/structures/labelstructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](../../../g1ant.language/structures/textstructure.md)  | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
```

Comment:
```suggestion
| `errormessage` | [text](../../../G1ANT.Language/Structures/TextStructure.md)  | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
```